src/index.js

```diff
@@ -277,25 +323,30 @@ OAuth.authorizeExplicitly = function(authURL, tokenURL, clientId, clientSecret,
 		};
 	win = Ti.UI.createWindow(AUTH_WINDOW_OPTIONS);
 	spinner = Ti.UI.createActivityIndicator({
-		zIndex : 1,
-		height : 50,
-		width : 50,
-		hide : true,
-		style : Ti.UI.ActivityIndicatorStyle.DARK
+		zIndex: 1,
+		height: 50,
+		width: 50,
+		hide: true,
+		style: Ti.UI.ActivityIndicatorStyle.DARK
 	});
-	webview = Ti.UI.createWebView({
-		width : '100%',
-		height : '100%',
-		url : buildURL(authURL, {
+	webViewDetails = {
+		width: '100%',
+		height: '100%',
+		url: buildURL(authURL, {
 			response_type: 'code',
 			client_id: clientId,
 			redirect_uri: CALLBACK_URL,
-			//scope: 'scope', // FIXME Allow user to pass in scopes?
+			scope: scope,
 			approval_prompt: 'force',
 			btmpl: 'mobile',
 			state: state
 		})
-	});
+	};
+	if (OAuth.ignoreSslError.get('value')) {
```

Comment:
Same as above, just put this in the original object hash above, no need for a special if block.